packages/legacy-client/src/ontology-runtime/ontologyProvider/parseStreamedResponse.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
Is this file in the right package or should it be in packages/api?

Reply:
This was specifically for parsing partial timeseries responses when loading a lot of data for results as they're available, probably fine to have that here

Reply:
Here is fine for now.